site/gulpfile.js

```diff
@@ -1,15 +1,18 @@
 var gulp = require('gulp');
 var elixir = require('laravel-elixir');
+var argv = require('yargs').argv;
 
 elixir.config.assetsPath = 'source/_assets';
 elixir.config.publicPath = 'source';
 
 elixir(function(mix) {
+	var env = argv.e || argv.env || 'local';
+
     mix.sass('main.scss')
-        .exec('jigsaw build', ['./source/*', './source/**/*', '!./source/_assets/**/*'])
+        .exec('jigsaw build --env ' + env, ['./source/**/*', '!./source/_assets/**/*'])
```

Comment:
Does this work without the `=` after the option name? If so that's neat, didn't realize it was optional, wonder what the best practice is?